src/styles/antisnake.scss

```
@@ -0,0 +1,33 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+
+  @media (max-width: 899px) {
+    grid-template-columns: 1fr 1fr ;
+  }
+
+  @media (max-width: 599px) {
+    grid-template-columns: 1fr;
+  }
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-width: 300px;
+    height: 300px;
+
+    @for $index from 0 through 5 {
+      &:nth-child(#{$index + 1}) {
+        $cell-color: 255 - 51 * $index;
+        background: rgb($cell-color, 0, 0);
+      }
+
+      &:nth-child(#{$index + 1})::before {
+        content: "#{$index + 1}";
+        color: #fff;
```

create vars for all colors

```suggestion
        color: $c-white;
```